### PR TITLE
New hint dialog of right contents drawer - on first launch

### PR DESCRIPTION
**Pull request by EladKeyshawn**

### CODE REVIEW

File: android/src/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 1]
```diff
@@ -1092,6 +1108,26 @@ public boolean openArticleFromBookmark(String bookmarkTitle) {
...
+  private void contentsDrawerHint() {
+    mLeftDrawerLayout.postDelayed(new Runnable() {
+      @Override
+      public void run() {
+        mLeftDrawerLayout.openDrawer(Gravity.RIGHT);
+      }
+    }, 500);
+
+    AlertDialog.Builder builder = new AlertDialog.Builder(this);
+    builder.setMessage("You can swipe left to view the contents of this article ")
```
Comment (mhutti1) [inline android/src/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Localise

File: android/src/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 2]
```diff
@@ -554,7 +557,20 @@ private KiwixWebView newTab() {
...
   private KiwixWebView newTab(String url) {
     KiwixWebView webView = new KiwixWebView(KiwixMobileActivity.this);
-    webView.setWebViewClient(new KiwixWebViewClient(KiwixMobileActivity.this, mLeftArrayAdapter));
+    webView.setWebViewClient(new KiwixWebViewClient(KiwixMobileActivity.this, mLeftArrayAdapter){
+      @Override
+      public boolean shouldOverrideUrlLoading(WebView view, String url) {
+        // onClick
+        if (isFirstRun) {
+          contentsDrawerHint();
+          SharedPreferences.Editor editor = getSharedPreferences("PREFERENCES", MODE_PRIVATE).edit();
```
Comment (mhutti1) [inline android/src/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: Use    SharedPreferences settings = PreferenceManager.getDefaultSharedPreferences(this);
    SharedPreferences.Editor editor = settings.edit();

And in all other locations.
As that is what is used mostly in the app

File: android/src/org/kiwix/kiwixmobile/KiwixMobileActivity.java [hunk 3]
```diff
@@ -1092,6 +1108,26 @@ public boolean openArticleFromBookmark(String bookmarkTitle) {
...
+      }
+    }, 500);
+
+    AlertDialog.Builder builder = new AlertDialog.Builder(this);
+    builder.setMessage("You can swipe left to view the contents of this article ")
+        .setPositiveButton("Got it", new DialogInterface.OnClickListener() {
+          public void onClick(DialogInterface dialog, int id) {
+          }
+        })
+        .setTitle("Did you know?")
```
Comment (mhutti1) [inline android/src/org/kiwix/kiwixmobile/KiwixMobileActivity.java]: I dont think this title is necessary. Its a bit patronising of the user

**Comment by rashiq** [comment]
🎉 🎉 🎉

**Comment by EladKeyshawn** [comment]
@rashiq Hey man! long time no see... thanks

**Pull request merged**